Makefile

```diff
@@ -157,32 +157,27 @@ all: binaries
 android-dist: genconfig android
 
 # This is to be called within the docker image.
-docker-genassets: require-npm
+genassets: require-npm
```

Comment:
Not sure if this is an advantage, what's the main benefit of running this locally? On the other hand, the reason to run build tasks inside docker is that everyone has the same environment, the same version of node, npm and everything else, so it may look like we could skip docker but I think that having that guarantee is important, specially with node-npms's ecosystem which already caused *it-works-for-me* problems in the past.